recipes/pytest-randomly/meta.yaml

```
@@ -0,0 +1,43 @@
+{% set name = "pytest-randomly" %}
+{% set version = "3.5.0" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/pytest-randomly-{{ version }}.tar.gz
+  sha256: 440cec143fd9b0adeb072006c71e0294402a2bc2ccd08079c2341087ba4cf2d1
+
+build:
+  number: 0
+  skip: true   # [py<36]
+  script: {{ PYTHON }} -m pip install . -vv
+
+requirements:
+  host:
+    - pip
+    - python
+    - setuptools
+  run:
+    - importlib-metadata  # [py<38]
+    - pytest
+    - python
+
+test:
+  imports:
+    - pytest_randomly
+  commands:
+    - pip check
```

You can also test that the plugin is found by pytest:
```suggestion
    - pip check
    - py.test --traceconfig | grep pytest-randomly-{{ version }}
```